### PR TITLE
MBC Last Front End Spanish Translations

### PR DESCRIPTION
**Pull request by MocanaAtBroad**
This PR includes Angular es.json file updates that were not covered by previous PRs.
Includes edits by Tania and Marco. Everything is now translated (woohoo!)

### CODE REVIEW

File: ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json [hunk 1]
```diff
@@ -6,48 +6,48 @@
...
+        "MRS": "logo de Metastasis Research Society",
+        "Sisters": "logo de Sisters R Us Circle of Survivors",
+        "HMN": "logo de Health Ministries Network",
+        "Min": "logo de Minority Coalition",
+        "MBCCanada": "logo de Metastatic Breast Cancer Canada"
     },
     "Toolkit": {
         "Common": {
-            "Organization": "Cuenten Conmigo"
+            "Organization": "Count Me In"
```
Comment (ssettipalli) [inline ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json:50]: Assuming  we want this in english but want to confirm just incase

File: ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json [hunk 2]
```diff
@@ -121,10 +121,10 @@
         },
         "AboutUs": {
             "PageHeader": {
-              "Pt1": "El Metastatic Breast Cancer Project es parte de ",
-              "Pt2": "una organización sin fines de lucro que atrae a pacientes e investigadores para juntos acelerar descubrimientos en la investigación del cáncer."
+                "Pt1": "El Metastatic Breast Cancer Project es parte de ",
+                "Pt2": "una organización sin fines de lucro que atrae a pacientes e investigadores para juntos acelerar descubrimientos en la investigación del cáncer."
             },
-          "PageHeaderLink": "Visite el sitio web <em>Cuenten Conmigo</em>",
+            "PageHeaderLink": "Visite el sitio web <em>Count Me In</em>",
```
Comment (ssettipalli) [inline ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json:127]: Assuming  we want this in english but want to confirm just incase

File: ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json (hunk 1 above)
Comment (MocanaAtBroad) [reply ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json:50]: That is a good question, Sampath. Tania specifically asked me how it was used and I explained that it is used in most place in Angular that would just show the title. So this was her informed choice.

**Review by ssettipalli — APPROVED** [review]
looks good to me

**Pull request merged**